Community.md

```diff
@@ -0,0 +1,47 @@
+# Community Continuity Guidelines v2.0.0
+
+As the creators, and maintainers of this project. We want to ensure that the project lives and continues to grow. Not blocked by any singular person's computer time.
```

Comment:
`As the creators and maintainers of this project, we want to ...`

You could also change the second `.` to a `,` if you are okay with a 28 word sentence.